posydon/binary_evol/DT/step_detached.py

```diff
@@ -857,12 +856,12 @@ def get_MESA_labels(list_for_matching):
             else:
                 print(
                     "Matching completed unsuccessfully for star with properties: \n"
-                    f'mass = {star.mass:.3f}',
-                    f'log_R = {star.log_R:.3f}',
-                    f'center_he4 = {star.center_he4:.4f}',
-                    f'surface_he4 = {star.surface_he4:.4f}',
-                    f'surface_h1 = {star.surface_h1:.4f}',
-                    f'he_core_mass = {star.he_core_mass:.3f}',
+                    f'mass = {star.mass:.3f}, ',
+                    f'log_R = {star.log_R:.3f}, ',
+                    f'center_he4 = {star.center_he4:.4f}, ',
+                    f'surface_he4 = {star.surface_he4:.4f}, ',
+                    f'\nsurface_h1 = {star.surface_h1:.4f}, ',
```

Comment:
Why shell we have an additional line break here? Look arbitrary to me.

Reply:
It formats the output in a cleaner way, so that the lines are not too long and run over into the next line.

Reply:
This will depend on the display and the length of the values. Therefore I'd either suggest to allow for automatic instead of fixed line breaks or to set all the sizes. At the moment a different person or depending on the exact value you may get an automatic line break just short before the hard coded one.